motors/tacho_motor_class.c

```diff
@@ -255,7 +251,7 @@ struct tacho_motor_state_item {
 
 static struct tacho_motor_value_names tacho_motor_states[NUM_TM_STATES] = {
 	[TM_STATE_RUNNING]	= { "running" },
-	[TM_STATE_RAMPING]	= { "ramping" },
+	[TM_STATE_STOPPED]	= { "stopped" },
```

Comment:
I'm OK with this until the stopping flag - the application knows that it sent the stop command, so it should expect `ramping`. It does tell you direction, but not really because you could have just set a new `speed_sp` and issued `run-forever`. Not sure what information `stopping` conveys. 
Similar argument for the passive breaking flag - we already know that from the `stop-command` value. By that argument, we'd get rid of `holding`, but I can justify keeping it because it is an *active* mode of the motor - ie the motor must have non-zero duty_cycle to maintain the hold